src/style.css

```diff
@@ -0,0 +1,106 @@
+
+body {
+  margin: 0;
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.block {
+  flex: 1 300px;
+  height: 300px;
+  font-size: 100px;
+  text-align: center;
+  font-family: Arial, sans-serif;
+  color: white;
+  line-height: 300px ;
+}
+
+@media screen and (max-width:900px) and (min-width:600px) {
```

Comment:
You do not need to specify max-width if you have the next breakpoint set, it will automatically be up to 900 cause then you make a request with min-width 900